integration-cli/docker_api_containers_test.go

```diff
@@ -1730,3 +1736,48 @@ func (s *DockerSuite) TestContainersApiCreateMountsCreate(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestContainersApiCreateMountsTmpfs(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
AFAIK yes, linux only in the current implementation?
https://github.com/docker/docker/blob/b8265e55508db99aea632033b2f5008f921b11e2/container/container_windows.go#L89-L93

IMO implementation for Windows should be another PR, but I understand we need to design the API with consideration for Windows in this PR.

CC @jhowardmsft